MIGRATION.md

```diff
@@ -0,0 +1,89 @@
+# Migrating to the `cloudant-node-sdk` library
+This document is to assist in migrating from the `nodejs-cloudant` (module: `cloudant`) to the newly supported [`cloudant-node-sdk`](https://github.com/IBM/cloudant-node-sdk) (package: `@ibm-cloud/cloudant`). Compatible with Javascript and TypeScript.
```

Comment:
Nit:
```suggestion
This document is to assist in migrating from the `nodejs-cloudant` (module: `cloudant`) to the newly supported [`cloudant-node-sdk`](https://github.com/IBM/cloudant-node-sdk) (package: `@ibm-cloud/cloudant`) that is supported in Javascript and TypeScript.
```

Reply:
This way `supported` word is used 2 times in a sentence. On the Hungarian Literature lesson taught me this is an invalid composition, better to use synonyms.
Okay, this is not a novel, but still. 
WDYT about this:
```suggestion
This document is to assist in migrating from the `nodejs-cloudant` (module: `cloudant`) to the newly supported [`cloudant-node-sdk`](https://github.com/IBM/cloudant-node-sdk) (package: `@ibm-cloud/cloudant`) that is applicable with Javascript and TypeScript.
```

Reply:
Good call. I'm not an English teacher though :)

Reply:
Yeah, I checked this word, it is rather used in legal context. So switched back to `compatible` currently in c706430

Reply:
Today I found an other word: `compliant` WDYT?

Reply:
Check it here: 583c25f

Reply:
Reverted this back: 40dd63a